src/modules/popComment.js

```diff
@@ -0,0 +1,59 @@
+import getAPIMovies from './requestAPIMovies.js';
+import likeIcon from '../images/red-heart-icon.png';
+
+const popComment = async (event) => {
+  const movies = await getAPIMovies();
+  const container = document.querySelector('.comments');
+  const overlay = document.createElement('div');
+  const span = document.createElement('span');
+  span.innerHTML = '<span class="close-icon">&times;</span>';
+  const popContent = document.createElement('div');
+  movies.forEach((movie) => {
+    if (parseInt(event, 10) === movie.id) {
+      popContent.innerHTML = `<div class="popContent">
+            <div class="col">
+                <img class="movie-image" src="${movie.image.original}" alt="Movie Poster">
+            </div>
+            <div class="wrapper">
+                <div class="col movie-summary">
+                    <p><b>${movie.name}</b> : ${movie.summary}</p>
+                </div>
+                    <ul class ="genres">
+                        <li class "genre1">${movie.genres[0]}</li>
+                        <li class "genre2">${movie.genres[1]}</li>
+                        <li class "genre3">${movie.genres[2]}</li>
+                    </ul>
+                <div class="col movies-likes">
+                    <div><img src="${likeIcon}" class="like-icon" alt="Red Heart Icon">
+                    <p class="likes-holder">${movie.rating.average}</p></div>
+                    <div class ="comments"><p>Comments (${movie.rating.average})</p></div>
+                </div>
```

Comment:
- [ ] Please implement functionality in the comment section so that the number of comments can be updating in real-time on user activity.